### PR TITLE
Removed broken links

### PR DESCRIPTION
**Pull request by alexture**
I removed broken links + the table of contents (since I'm planning on changing the structure very soon). This should be a pretty straightforward pull, the next one will be bigger! :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,7 +116,6 @@ This list contains tools, books, articles, blogs, courses and everything related
 - [Lexico](https://www.lexico.com/) - Provides a collection of dictionaries of English and Spanish, and curiosities about many words and languages (in the "Explore" section). It is produced by Oxford University Press.
 - [The Translator's Teacup](https://lingocode.com/translation-blog/) - Contains good tips for beginner translators.
 - [Speaking of Translation](https://speakingoftranslation.com/) - Run by two outstanding experts in the field of translation and localization. You can listen to audio files covering their tips and opinions on the freelance translator's work.
-- [Between Translations](http://foxdocs.biz/BetweenTranslations/) - Run by Jayne Fox, a German into English translator. It talks about continuing professional development (CPD) for translators and the use of technology (especially CAT tools/TEnTs) to support their work.
```
Comment (mbiesiad) [inline README.md:134]: Nice job! Error 500 at this moment there

File: README.md [hunk 2]
```diff
@@ -56,9 +44,6 @@ This list contains tools, books, articles, blogs, courses and everything related
 - [Copy Translator](https://github.com/CopyTranslator/CopyTranslator) - An app for Windows, Mac and Linux that automatically translate and adapt a text when copied in the clipboard.
 - [LibreTranslate](https://github.com/uav4geo/LibreTranslate) - 100% Self-Hosted web app to translate text, based on Argos Translate.
 
-### i18n manager
-- [Electronjs i18n](https://www.electronjs.org/apps/i18n-manager) - I18n manager for Windows.
```
Comment (mbiesiad) [inline README.md:60]: Agree, error 404 now there - well done!

File: README.md [hunk 3]
```diff
@@ -4,18 +4,6 @@
 
 This list contains tools, books, articles, blogs, courses and everything related to translations ([t9n](https://en.wikipedia.org/wiki/Translation) - translation, [l10n](https://en.wikipedia.org/wiki/Language_localisation) - localization, [g11n](https://en.wikipedia.org/wiki/Language_localisation#Globalisation_versus_localisation) - globalization, [m17n](https://www.definify.com/word/multilingualization#:~:text=Noun,into%2C%20or%20for%20multiple%20languages) - multilingualization, [i18n](https://en.wikipedia.org/wiki/Language_localisation#Globalisation_versus_localisation) - internationalization, synonymous with globalization).
 
-## Contents
```
Comment (mbiesiad) [inline README.md:7]: I suppose we should leave this now (yet), that's valid, correct internal links :P

**Review by mbiesiad — CHANGES_REQUESTED** [review]
Agree about err 404 and 500, great work! But I suppose that internal links are useful, until we don't have new structure, it's correct IMO

**Comment by mbiesiad** [comment]
Hi @alexture Once again, thanks so much! In general I'm with you, agree about provided changes, but before we make new structure, in my opinion we should have ToC with internal links - many people are using these resources day after day. Thank you for understanding, best wishes

**Comment by alexture** [comment]
Hi! I _think_ I fixed this but am still trying to figure out GitHub. Hopefully my latest commit added back the table of contents and still had the broken links removed :)

**Comment by mbiesiad** [comment]
Hi. All is fine, merged. 🎉 Thanks a lot! ✨

**Pull request merged**